src/resources/Guest.ts

```diff
@@ -119,12 +171,30 @@ interface GetResponseInterface {
   }
 }
 
+interface CheckinResponseInterface {
+  data: {
+    guest: GuestInterface
+  }
+}
+
+interface RestoreResponseInterface {
+  data: {
+    guest: GuestInterface
+  }
+}
+
 interface CreateResponseInterface {
   data: {
     guest: GuestInterface
   }
 }
 
+interface CreateRecommendationResponseInterface {
+  data: {
+    guest: GuestInterface
+  }
+}
```

Comment:
Probably we can use `GetResponseInterface` everywhere instead of having duplicate ones. If they start to differ, then we create a different, dedicated interface